packages/experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -91,4 +110,15 @@ export interface ISelectedItemsListProps<T> extends React.ClassAttributes<any> {
    * A callback on whether this item can be removed
    */
   canRemoveItem?: (item: T) => boolean;
+
+  /** Map of callback functions related to row drag and drop functionality. */
```

Comment:
is the comment correct?

Reply:
It's not actually a map. Fixed in the latest update.